app/apps/client/admin/apps.js

```diff
@@ -240,7 +240,7 @@ Template.apps.events({
 		instance.startAppWorking(app.id);
 
 		try {
-			const { status } = await Apps.installApp(app.id, app.marketplaceVersion);
+			const { status } = await Apps.updateApp(app.id, app.marketplaceVersion);
```

Comment:
I wonder if it is fesiable too handle `click .js-update` and `click .js-update` with two handlers here like what you do in the `admin/marketspace.js`. What will happen if you click an `install` button, is it working as expected?

Reply:
I think it would be better if I simply removed the `install` handler from the app admin page. I saw no install button in there as it only lists already installed apps and displays the `update` button when needed. I didn't notice at first, tbh

Reply:
okay, it makes sense. Thanks!

Reply:
Solved